packages/config/src/index.ts

```diff
@@ -0,0 +1,97 @@
+import * as enums from './enums';
+import { Autocomplete } from './Autocomplete'
+import { Recommendation } from './Recommendation'
+import { Search } from './Search'
+import { Content } from './Content'
+import { Method } from '@findify/sdk/types/request'
+
+type Config = {
+  /** Merchant API key */
+  key: string
+
+  /** Merchant ID, required to connect Chrome DevTools */
+  merchant_id: number
+
+  /** Store status */
+  status: enums.StoreStatus
+
+  /** Store platform. Generic by default */
+  platform: {
+    [key in enums.Platform]?: boolean
+  }
+
+  /** Disable event tracking in analytics */
+  analytics?: {
+    [key in enums.AnalyticsEventKey]?: false
+  }
+
+  api: {
+		method: Method
```

Comment:
@Bitaru looks like indentation is broken

Reply:
wtf...